src/components/Thanks/MyKiva/OptInModule.vue

```diff
@@ -0,0 +1,261 @@
+<template>
+	<transition-group name="collapse" tag="div">
+		<template v-if="!newConsentAnswered">
+			<div class="module-container">
+				<h2>{{ title }}</h2>
+				<div class="tw-flex tw-items-center tw-justify-center">
+					<BorrowerImage
```

Comment:
Why is this using `BorrowerImage` instead of `KvUserAvatar`? That might be where some of the rendering differences I'm seeing are, such as the little dark line around the rounded edge of the image. Also these 1-3 borrower avatars need a shadow. For the shadow, you could use `tw-shadow`.

Current:
![image](https://github.com/user-attachments/assets/c14fbb49-3fd1-455b-933b-780f585caba2)

Figma:
![image](https://github.com/user-attachments/assets/25a9014b-eb30-4770-86cd-5d510057e706)